src/main/java/org/openrewrite/xml/liberty/ChangeNamespaceValue.java

```diff
@@ -0,0 +1,152 @@
+/*
+ * Copyright 2023 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.xml.liberty;
+
+import org.openrewrite.*;
+import org.openrewrite.internal.ListUtils;
+import org.openrewrite.xml.XPathMatcher;
+import org.openrewrite.xml.XmlVisitor;
+import org.openrewrite.xml.tree.Xml;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import org.openrewrite.internal.lang.NonNull;
+import org.openrewrite.internal.lang.Nullable;
+
+import lombok.EqualsAndHashCode;
+import lombok.Value;
+
+@Value
+@EqualsAndHashCode(callSuper = true)
+public class ChangeNamespaceValue extends Recipe {
```

Comment:
By chance(?) we also got a request to add a recipe to change the namespace in https://github.com/openrewrite/rewrite/pull/3462.
We're working out the differences and will probably adopt one of the other there, such that we don't have to add it here.

Reply:
Haha that's interesting. It would make sense to have the code in the core rewrite repo then. I can update my code to point to it once it's merged.

Reply:
Yes we were working out how to get you credited in the Git history with creating that recipe in openrewrite/rewrite, but maybe the easiest way would be for you to open a pull request there with that one recipe, if you don't mind.

Reply:
If you don't care for credits we can of course copy your class there ourselves; and update accordingly here as well.

Reply:
No problem. I'll have the pull request out soon.